zpm.t

```diff
@@ -31,8 +31,8 @@ Subsequentially add a new item to the list.
 List command show list of installed plugins.
 
     $ ./zpm list
-    zsh-users/zsh-syntax-highlighting
-    rupa/z
+    zsh-users/zsh-syntax-highlighting .*
```

Comment:
We should probably add tests for this but for now I want to get this merged ASAP to avoid huge git conflicts.